wiki_trac_rst_convert.py

```diff
@@ -71,8 +105,8 @@ def _trac_to_github_wiki_links(text: str):
         '`wiki:(.+?)`:trac:',
 
         # TracWiki markup:
-        '`\[wiki:"?([^ ]+?)"?]`:trac:',
-        '\[wiki:"?([^ ]+?)"?]',
+        r'`\[wiki:"?([^ ]+?)"?]`:trac:',
```

Comment:
sorry, I forgot to answer to your comment about `[`. Yes. Using raw format is best.